configs/segnext/segnext.yml

```diff
@@ -0,0 +1,103 @@
+Collections:
+- Name: SegNeXt
+  Metadata:
+    Training Data:
+    - ADE20K
+  Paper:
+    URL: https://arxiv.org/abs/2209.08575
+    Title: 'SegNeXt: Rethinking Convolutional Attention Design for Semantic Segmentation'
+  README: configs/segnext/README.md
+  Code:
+    URL: https://github.com/open-mmlab/mmsegmentation/blob/v0.31.0/mmseg/models/backbones/mscan.py#L328
+    Version: v0.31.0
+  Converted From:
+    Code: https://github.com/visual-attention-network/segnext
+Models:
+- Name: segnext_mscan-t_1x16_512x512_adamw_160k_ade20k
+  In Collection: SegNeXt
+  Metadata:
+    backbone: MSCAN-T
+    crop size: (512,512)
+    lr schd: 160000
+    inference time (ms/im):
+    - value: 19.09
+      hardware: A100
+      backend: PyTorch
+      batch size: 1
+      mode: FP32
+      resolution: (512,512)
+    Training Memory (GB): 17.88
+  Results:
+  - Task: Semantic Segmentation
+    Dataset: ADE20K
+    Metrics:
+      mIoU: 41.5
+      mIoU(ms+flip): 42.59
+  Config: configs/segnext/segnext_mscan-t_1x16_512x512_adamw_160k_ade20k.py
+  Weights: https://download.openmmlab.com/mmsegmentation/v0.5/segnext/segnext_mscan-t_1x16_512x512_adamw_160k_ade20k/segnext_mscan-t_1x16_512x512_adamw_160k_ade20k_20230210_140244-05bd8466.pth
+- Name: segnext_mscan-s_1x16_512x512_adamw_160k_ade20k
+  In Collection: SegNeXt
+  Metadata:
+    backbone: MSCAN-S
+    crop size: (512,512)
+    lr schd: 160000
+    inference time (ms/im):
+    - value: 23.66
+      hardware: A100
+      backend: PyTorch
+      batch size: 1
+      mode: FP32
+      resolution: (512,512)
+    Training Memory (GB): 21.47
+  Results:
+  - Task: Semantic Segmentation
+    Dataset: ADE20K
+    Metrics:
+      mIoU: 44.16
+      mIoU(ms+flip): 45.81
+  Config: configs/segnext/segnext_mscan-s_1x16_512x512_adamw_160k_ade20k.py
+  Weights: https://download.openmmlab.com/mmsegmentation/v0.5/segnext/segnext_mscan-s_1x16_512x512_adamw_160k_ade20k/segnext_mscan-s_1x16_512x512_adamw_160k_ade20k_20230214_113014-43013668.pth
```

Comment:
this link is broken

Reply:
> this link is broken

OK, I also need to transfer keys of those four checkpoints because `DWConv` module is deleted, so I would update new checkpoints later.

Reply:
> > this link is broken
> 
> OK, I also need to transfer keys of those four checkpoints because `DWConv` module in `mscan.py` is deleted, so I would update new checkpoints later.

Done.